src/libraries/Microsoft.Win32.Registry/src/System/Security/AccessControl/RegistrySecurity.cs

```diff
@@ -95,7 +95,7 @@ internal RegistrySecurity(SafeRegistryHandle hKey, string name, AccessControlSec
         {
         }
 
-        private static Exception _HandleErrorCode(int errorCode, string name, SafeHandle handle, object context)
+        private static Exception? _HandleErrorCode(int errorCode, string name, SafeHandle handle, object context)
```

Comment:
Arguments nullability should match with the `ExceptionFromErrorCode` delegate declaration
`protected internal delegate Exception ExceptionFromErrorCode(int errorCode, string? name, SafeHandle? handle, object? context);`

Reply:
and return type nullability